test/acceptance/test/add_tests.go

```diff
@@ -736,9 +738,29 @@ var _ = Describe("Weave GitOps App Add Tests", func() {
 			Expect(folderOutput).Should(ContainSubstring("apps"))
 			Expect(folderOutput).Should(ContainSubstring("targets"))
 		})
+
+		By("When I check for list of commits for app1", func() {
+			commitList1, _ = runCommandAndReturnStringOutput(fmt.Sprintf("%s app %s get commits", WEGO_BIN_PATH, appName1))
+		})
+
+		By("Then I should see the list of commits for app1", func() {
+			Eventually(commitList1).Should(MatchRegexp(`COMMIT HASH\s*CREATED AT\s*AUTHOR\s*MESSAGE\s*URL`))
+			Eventually(commitList1).Should(MatchRegexp(`[\w]{7}\s*202\d-[0,1][0-9]-[0-3][0-9]\s*[0-2][0-9]:[0-5][0-9]:[0-5][0-9]`))
+			Eventually(commitList1).Should(MatchRegexp(`[\w]{7}\s*202\d-[0,1][0-9]-[0-3][0-9]\s*[0-2][0-9]:[0-5][0-9]:[0-5][0-9]`))
```

Comment:
Can we put these expressions in variables to give the reader an idea of what they are checking for?

Reply:
The message under `By("Then I should see the list of commits for app1")` should be enough to tell the reader what we're looking for. The rest is regexp to match that outcome.

Reply:
Pretty tough to debug those regular expressions to figure out what they are looking for specifically.